parcels/particleset.py

```diff
@@ -172,41 +199,33 @@ def __getattr__(self, name):
         name : str
             Name of the property
         """
-        for v in self.particledata.ptype.variables:
-            if v.name == name:
-                return getattr(self.particledata, name)
+        if name in self._data:
+            return self._data[name]
         if name in self.__dict__ and name[0] != "_":
             return self.__dict__[name]
         else:
             return False
```

Comment:
I think that this can be reduced to: `return self._data[name]` (see other comment)

Reply:
Fixed in d0c5537c0abd410f12dbc9063e50a12a437cd526